related-projects.md

```diff
@@ -1,7 +1,36 @@
+
 # Overview Related Projects
 
 Please add your own projects and insights you have about others.
 
-| Name/Link | Description | Interesting ideas |
-| --- | --- | --- |
-| [Concordis](https://concordis.io/) |  | <ul><li>Re-Use twitter identity verification</li><li>Challenge thought leaders with argument maps on twitter.</li></ul> |
+| Name/Link | Description |
```

Comment:
Why did you delete the `Interesting Ideas` column?

Reply:
As mentioned on Slack: "I removed the 'interesting insights' column as it is way too subjective. Won't be maintainable. Instead,  I feel that should go in the description column."